examples/scaling-criteo/03-Training-with-HugeCTR.ipynb

```diff
@@ -311,9 +311,12 @@
     "        top_names=[\"loss\"],\n",
```

Comment:
<p>instead of writing <code>model.py</code> the code should just be run directly in the cell, and instead of using <code>import time</code> just use <code>%%time</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>